client/src/components/Feed/NestedComments.js

```diff
@@ -74,62 +108,144 @@ const NestedComments = ({ isAuthenticated, comment }) => {
     setReply("");
   };
 
-  const renderReplyInput = showReply ? (
-    <div className="reply-input">
-      {renderAvatar}
-      {isAuthenticated ? (
-        <AutoSize
-          placeholder={"Write a reply..."}
-          onPressEnter={handleReply}
-          onChange={(e) => setReply(e.target.value)}
-          value={reply}
-        />
-      ) : (
-        ""
-      )}
-    </div>
-  ) : (
-    ""
-  );
-
-  const renderReply = () => {
-    const style = showReply ? clickedTextStyle : {};
-    return (
-      <span
-        style={style}
-        onClick={() => setShowReply(!showReply)}
-        key="comment-nested-reply-to"
-      >
-        Reply
-      </span>
-    );
-  };
-  const renderTimeStamp = <span>1w</span>;
+  // const renderReplyInput = showReply ? (
+  //   <div className="reply-input">
+  //     {renderAvatar}
+  //     {isAuthenticated ? (
+  //       <AutoSize
+  //         placeholder={"Write a reply..."}
+  //         onPressEnter={handleReply}
+  //         onChange={(e) => setReply(e.target.value)}
+  //         value={reply}
+  //       />
+  //     ) : (
+  //       ""
+  //     )}
+  //   </div>
+  // ) : (
+  //   ""
+  // );
 
-  //TODO: Add comment replies, like button and number of likes.
-  // const commentActions = [
-  //   renderTimeStamp,
-  //   renderLikeButton(),
-  //   renderReply(),
-  //   renderNumLikes(),
-  //   renderReplyInput,
-  // ];
+  // const renderReply = () => {
+  //   const style = showReply ? clickedTextStyle : {};
+  //   return (
+  //     <span
+  //       style={style}
+  //       onClick={() => setShowReply(!showReply)}
+  //       key="comment-nested-reply-to"
+  //     >
+  //       Reply
+  //     </span>
+  //   );
+  // };
 
   const nestedComments = (comment.children || []).map((comment) => {
     return <NestedComments comment={comment} key={comment._id} />;
   });
 
+  const handleCommentEdit = (e) => {
+    setEditedComment(e.target.value);
+  };
+
+  const handleSubmit = async () => {
+    let response;
+    const postId = comment.postId;
+    const commentId = comment._id;
+    const payload = { content: editedComment };
+
+    if (isAuthenticated && comment.author.id === user.id) {
+      const endPoint = `/api/posts/${postId}/comments/${commentId}`;
+
+      try {
+        response = await axios.patch(endPoint, payload);
+        if (response && response.data) {
+          dispatchPostAction(
+            SET_COMMENT,
+            "commentId",
+            commentId,
+            "comment",
+            response.data.content,
+          );
+          setEditComment(!editComment);
+        }
+      } catch (error) {
+        console.log({ error });
```

Comment:
can either remove or use `console.error` too, maybe we want to show that something went wrong to the user? 
i often ask this to designers to be more specific about error states in the UI since this is often just left out